.github/workflows/cypress-tests.yml

```
@@ -0,0 +1,161 @@
+    
+    steps:
+    - name: Checkout
+      uses: actions/checkout@v4
+      
+    - name: Setup Node.js
+      uses: actions/setup-node@v4
+      with:
+        node-version: '18'
+        cache: 'npm'
+        cache-dependency-path: frontend/package-lock.json
+        
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)